JoodHajeer/assignment1/Q1

```diff
@@ -4,66 +4,68 @@
 #include <string>
 #include <map>
 #include <algorithm>
+#include <cctype>
 using namespace std;
-map<string,int> mp; //maps a string to its frequency in the sentence
-void toLower(string &s){ //change all capital letters in s to lower letters
-	for(int i=0;i<s.size();i++)
-		if(s[i]>='A'&&s[i]<='Z')
-			s[i]+='a'-'A';
+
+void toLower(string &s) { //change all capital letters in s to lower letters
```

Comment:
Minor comment, it's a better to use pointers instead of references for parameters you want to modify. It's a bit more cumbersome to write all the derreferences, but it makes crystal clear for the caller that the variable will be modified, compare:
toLower(s) vs toLower(&s)